### PR TITLE
removed logic to add +1 to phone numbers 

### PR DESCRIPTION
**Pull request by greggraf**
in expectation that the server will provide. Updated logic in href to remove space in phone numbers

### CODE REVIEW

**Comment by nickoneill** [comment]
Thanks! This'll make it easy to roll this out when we change the backend.

Test fails on a js error, fix looks like this: https://github.com/5calls/5calls/tree/pr/305

File: static/js/components/contact.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = (c, state, prev, send) => {
           <h3 class="call__contact__field-offices__header">${t('contact.localOfficeNumbers')}</h3>
           <ul class="call__contact__field-office-list">
             ${c.field_offices.map(office => html`
-              <li><a href="tel:+1${office.phone.replace(/-/g, '')}">${office.phone}</a> ${cityFormat(office,c)}</li>
+              <li><a href="tel:${office.phone.replace(/-| /g, '')}">${office.phone}</a> ${cityFormat(office,c)}</li>
```
Comment (oderby) [inline static/js/components/contact.js]: Visually, it looks cluttered/clumsy to *display* the phone number with the "+1 " prefix, so I think we should remove it. Perhaps we can strip the prefix for the display only? Something like `office.phone.replace(/^\+\d* /, '')` here and below.

**Comment by greggraf** [comment]
@oderby - I updated the tests to verify the formatting of both the href and the display text and changed the code with what I think you were suggesting.

**Review by oderby — APPROVED** [review]
lgtm. Tested that this works against current backend, so I think this is safe to merge as is, independent of backend changes.

**Pull request merged**